app/modules/settings/views/templates/display_settings.hbs

```diff
@@ -0,0 +1,35 @@
+<h2 class="layout-title">{{I18n 'display'}}</h2>
+
+<section class="entities">
+  <h3>{{i18n 'works'}} / {{i18n 'series'}} / {{i18n 'authors'}} {{i18n 'lists'}}</h3>
+  <a class="selector">
+    <span class="label">{{I18n 'current display'}} : </span>
+    <div class="{{#if entitiesCascade}}shown{{/if}}">{{icon 'th-large'}} {{I18n 'cascade'}}</div>
+    <div class="{{#if entitiesTable}}shown{{/if}}">{{icon 'align-justify'}} {{I18n 'table'}}</div>
+  </a>
+  <ul id="entitiesDisplayOptions">
+    <li displayType="entitiesDisplay" id="entitiesCascade">
```

Comment:
as that's a non standard attribute, it should be define with the [`data` prefix](https://developer.mozilla.org/fr/docs/Web/HTML/Global_attributes/data-*)
```suggestion
    <li data-display-type="entitiesDisplay" id="entitiesCascade">
```

same for the 3 other occurrences